### DISCUSSION TITLE
2024-05-01 Weekly Meeting 157 Notes/Agenda

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#464] [Weekly Meetings]
Weekly Meeting 157 is happening on Wednesday, 1 May 2024 from 9:00 AM PDT (17:00 UTC) - 10:00 AM PDT (18:00 UTC)
(Public holiday in some parts) 

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting in the omi-weekly-meeting channel of the [OMI Discord](https://discord.gg/jVaUteJyh7).
We regularly use the # omi-meeting-chat channel for text chat.

### Agenda
Check in, welcome
Fun finds!
Working groups check in
Demo's

**Fun finds:** 

TechnoWizard Anarchist: [Meta opening up their OS](https://about.fb.com/news/2024/04/introducing-our-open-mixed-reality-ecosystem/), ThrillSeeker did a great breakdown: https://youtu.be/Ck0P8HfR_OE?si=6DbxDKeojRMmOmJx
jin: [Open Metaverse Research Group](https://omrg.org/w/index.php?title=Main_Page) - interesting community that mentions OMI in their wiki

**Ongoing conversations**

- Lightning talks - to plan next: a hackathon to let people practice through their interview questions, and an end to end kind of sweep to find out the resources on how to do this - by practising with locals first - streamline the resources required. (Also good for thinktank conversations)
- Collaborative Library Project - potential to link up with Virtual Worlds Museum?
- "if a blogpost was a 3D object, what would it look like?" - hackathon potential!
- Linux Foundation collaboration / OMF revival - send email (hopefully before this meeting)

Last week's notes: https://github.com/omigroup/omigroup/discussions/463
Meeting recordings are at https://mee6.xyz/en/dashboard/887789163045335081/record

**Comment by indiebio** [comment]
TODO: reach out to [Open Metaverse Research Group](https://omrg.org/w/index.php?title=Main_Page) and see what synergies can form, how we can help each other - humbletim to reach out for starters

**Comment by indiebio** [comment]
Meta opening up their OS: member sentiments:
- Love that the Meta OS is opening up.  "The Android of mixed reality"
- I think by stating their commitment to Open Standards, they are taking a leadership position with their hardware.
- interoperable asset inventories much needed.

**Comment by mrmetaverse** [comment]
can we gamify/incentivize turning these into more checks? maybe use some of our funds to issue bounties? 

[![image](https://github.com/omigroup/omigroup/assets/63426722/c695a690-9944-4148-a549-b3910a09f3e3)](https://github.com/omigroup/gltf-extensions?tab=readme-ov-file#extension-support-table)

[![image](https://github.com/omigroup/omigroup/assets/63426722/9ad57451-8eb9-4fcf-87cf-7eb587246cb7)](https://github.com/omigroup/gltf-extensions?tab=readme-ov-file#extension-support-table)

**Reply by mrmetaverse** [reply]
![image](https://github.com/omigroup/omigroup/assets/63426722/bfd91655-a1e9-4df8-af52-e74c39923707)
